build/cleanup-images.yml

```diff
@@ -0,0 +1,55 @@
+# DESCRIPTION: 	
+# Removes old docker images.
+
+trigger: none
+pr: none
+
+variables:
+- template: build-variables.yml
+
+jobs:
+- job: deleteImage
+  displayName: 'Delete Images'
+  pool:
+    vmImage: $(WindowsVmImage)
+  steps:
+  - task: AzureCLI@2
+    displayName: 'Delete Images'
+    imputs:
+      azureSubscription: $(ConnectedServiceName)
+      scriptType: ps
+      scriptLocation: InlineScript
+      inlineScript: |
+        $versions = "stu3","r4","r5"
+        $tagList = az acr repository show-tags --name $(azureContainerRegistryName) --repository r4_fhir-server
+
+        foreach ($tag in $tagList) {
+          $tagParts = $tag.Split('-')
+          if ($tagParts.Count -gt 1) {
+            try {
+              $imageDate = [datetime]($tagParts[1].Substring(0,4) + '/' + $tagParts[1].Substring(4,2) + '/' + $tagParts[1].Substring(6,2))
```

Comment:
Is there another date we could use? Ideally I'd eventually want to switch to semver for tags too

Reply:
We can look up the image's last updated date, it just takes a few more steps.

Reply:
Changed to use the last updated date.